source/Halibut.Tests/Support/TestAttributes/LatestAndPreviousClientAndServiceVersionsTestCasesAttribute.cs

```diff
@@ -41,14 +41,18 @@ public static IEnumerable GetEnumerator(bool testWebSocket, bool testNetworkCond
                 {
                     serviceConnectionTypes.Remove(ServiceConnectionType.Polling);
                 }
+
+                var skipBackwardsCompatibilityTests = Environment.GetEnvironmentVariable("SKIP_BACKWARDS_COMPATIBILITY_TESTS") == "true";
```

Comment:
I leveraged an environment variable to skip these particular tests as it was reliable and easy to configure. If there is a preferred alternative approach, please let me know.

Reply:
Is this so they can be skipped on Windows 2012?
Don't we normally use attribute on the tests specifying categories, and then skip by category?

Reply:
> Is this so they can be skipped on Windows 2012?

Yes. I kept the environment variable name agnostic and focused on what it actually did.

> Don't we normally use attribute on the tests specifying categories, and then skip by category? Is there anything equivalent to that we could try here?

I'm not sure. This attribute basically sets up the test data for these parameterised tests. I don't think a particular set of test data can be assigned a specific category (but may be wrong).